virtual-hyperscript/hooks/attribute-hook.js

```diff
@@ -12,15 +12,21 @@ function AttributeHook(namespace, value) {
 }
 
 AttributeHook.prototype.hook = function (node, prop, prev) {
-    if (prev && prev.value === this.value) {
+    if (prev instanceof AttributeHook &&
```

Comment:
We aggressively avoid `instanceof` in virtual-dom to prevent errors across boundaries and duplication. Favor putting a `type` field on the attribute hook prototype and compare against that.